Frontend/src/pages/ProfileCompletion.jsx

```
@@ -0,0 +1,188 @@
+              onClick={() => toggleInterest(interest)}
+              onKeyDown={(e) => {
+                if (e.key === "Enter" || e.key === " ") {
+                  toggleInterest(interest);
+                }
+              }}
+            >
+              {interest}
+            </span>
+            ))}
+          </div>
+          <button onClick={handleNext} className="btn-primary">Next</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)